app/templates/signatures/ongoing.hbs

```diff
@@ -20,22 +20,30 @@
       @currentSorting={{this.sort}}
       @onChange={{fn (mut this.sort)}}
     />
+    <th>{{t "minister"}}</th>
     <Utils::ThSortable
       @label={{t "date-approved"}}
       @field="decision-activity.start-date"
       @currentSorting={{this.sort}}
       @onChange={{fn (mut this.sort)}}
     />
     <th>{{t "status"}}</th>
+    <th></th>
   </:header>
   <:body as |signFlow|>
     {{#let signFlow.signSubcase.signMarkingActivity as |signMarkingActivity|}}
       {{#let signMarkingActivity.piece as |piece|}}
-        <td>
+        <td class="au-u-flex au-u-flex--spaced-tiny">
```

Comment:
the flex here messes with the rows for me.
![image](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/assets/22245223/559b3e64-5c90-49f3-8837-04600cf45d32)
Longer titles mean a large column size, but lots of whitespace
![Uploading image.png…]()

Reply:
with pill
![image](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/assets/22245223/b15c5402-b06f-4c04-8679-eba7d4121699)

Reply:
removed the flex from the column, looks better